backend/internal/usecase/transaction.go

```diff
@@ -228,19 +229,20 @@ func (uc *TransactionUsecase) QueryTransactionDetailsByTxHash(ctx context.Contex
 			}
 		} else if tse.TxStatus.TxStatusVal == data.TxStatusValUnshieldedinclusion {
 			txStatus = UnshieldedInclusion
-			effectiveInclusionTime = tse.DecryptedTxCreatedAtUnix
+			effectiveInclusionTime = tse.DecryptedTxUpdatedAtUnix
 			inclusionDelay = effectiveInclusionTime - tse.CreatedAt.Time.Unix()
-			if tse.Slot.Valid {
```

Comment:
Same here, since we are not returning the correct slot and block numbers, it would be preferable to set them to null.